CODEOWNERS

```diff
@@ -62,26 +62,51 @@ Jenkinsfile @cilium/ci
 Jenkinsfile.nightly @cilium/ci
 monitor/ @cilium/monitor
 monitor/payload @cilium/api
+pkg/annotation @cilium/kubernetes
 pkg/apierror/ @cilium/api
 pkg/apipanic/ @cilium/api
 pkg/apisocket/ @cilium/api
 pkg/bpf/ @cilium/bpf
+pkg/byteorde/ @cilium/bpf
+pkg/client @cilium/api
+pkg/comparator @cilium/bpf
 pkg/completion/ @cilium/proxy
+pkg/components/ @cilium/bpf
+pkg/controller @cilium/agent
+pkg/counter @cilium/bpf
+pkg/datapath @cilium/bpf
+pkg/debugdetection @cilium/agent
+pkg/defaults @cilium/agent
 pkg/endpoint/ @cilium/endpoint
+pkg/endpointmanager/ @cilium/endpoint
 pkg/envoy/ @cilium/proxy
+pkg/fqdn/ @cilium/agent
 pkg/health/ @cilium/health
+pkg/identity @cilium/policy
 pkg/ipcache/ @cilium/ipcache
 pkg/k8s/ @cilium/kubernetes
 pkg/kafka/ @cilium/proxy
 pkg/kvstore/ @cilium/kvstore
+pkg/labels @cilium/policy
+pkg/launcher @pkg/agent
+pkg/lock @pkg/agent
 pkg/logging/ @cilium/cli
+pkg/mac @cilium/bpf
 pkg/maps/ @cilium/bpf
+pkg/metrics @cilium/agent
```

Comment:
My reasoning was that the metrics are only relevant in the context of the agent, since we don't serve them anywhere else (we would collect them, however).  I've made a metrics team with the people that have touched metrics (so, you!) since I'm not sure what other selector we have ;)